### DISCUSSION TITLE
Possible trouble with 180th meridian

### DISCUSSION THREAD

**Discussion by s-lipatov** [osm-search/Nominatim#3378] [Q&A] [answered]
After installing Nominatim version 4.4, everything starts and works well. However, there seems to be a problem with polygonal features crossing the 180th meridian.
If we take the source:

PBF_URL=https://download.geofabrik.de/russia/far-eastern-fed-district-latest.osm.pbf

then after installation the request 

http://127.0.0.1:7070/lookup?osm_ids=R151231&limit=1&format=json&polygon_geojson=1

returns an empty array, for comparison 

https://nominatim.openstreetmap.org/lookup?osm_ids=R151231&limit=1&format=json&polygon_geojson =1 

returns expected.

If you look at the import tables:

SELECT * FROM place WHERE osm_id = 151231 AND osm_type = 'R' -- no results
SELECT * FROM planet_osm_rels WHERE id = 151231 -- ok

Is there a problem with osm2pgsql or what?

**Comment by mtmail** [comment]
I don't think the PDF file you downloaded contains anything beyond the 180 longitude. See the red border on https://download.geofabrik.de/russia/far-eastern-fed-district.html

![image](https://github.com/osm-search/Nominatim/assets/3727288/138d4163-e581-49ed-8481-8ae2f2ab12f7)

You can try to create your own extract from the planet. For example using https://extract.bbbike.org/ or https://docs.osmcode.org/osmium/v1.14.0/osmium-extract.html

Dealing with the meridian is notorious tricky with all tools. There's no good standards. You might need to create two extracts and combine them.

**Reply by lonvia** [reply]
There is data past the anti-meridian. You can see it, when you scroll to the far west past America.

I've had a look at the extract and it seems to be missing the ways along the anti-meridian. osm2pgsql cannot build the polygons without them. I've reported the issue with Geofabrik.

**Reply by woodpeck** [reply, marked as answer]
I have changed the extract boundaries on the Geofabrik download server to include things on the 180/-180 longitude. Files from 01 April 2024 or later should now contain these ways.

**Comment by s-lipatov** [comment]
Just checked, everything seems to be working correctly now. Thank you very much!
